.github/workflows/ci.yml

```
@@ -0,0 +1,35 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby-version:
+          - 3.0.6
+          - 3.1.4
+          - 3.2.2
```

Let's not fix the minor versions.
```suggestion
          - 3.0
          - 3.1
          - 3.2
```

---

Makes sense!